workflow/schemas/meeting.json

```diff
@@ -24,11 +24,11 @@
       "Description": "Add basic information about this meeting event",
       "widgets": [
         {
-          "label": "Event Type",
-          "description": "Select the type of event that is being submitted. Events may be \"talk\" or \"poster\"",
+          "label": "Event Kind",
```

Comment:
Event Type on user-facing text might be clearer, even if it's kind on the backend. What do you think?

Reply:
I honestly dont know. type is pretty abstract too though. maybe we should go with consistency now, and do a/b testing or poll people?

Reply:
Actually I take that back.. this would be a user changeable thing- its the workflow. The examples are likely what people will be using to build their own, and it'd be weird for it to be named differently.

Reply:
what if you call it item_type on the backend and Event Type on the frontend?

Reply:
I'm just trying to channel Jeff and product team here